src/wp-includes/class-wp-theme-json-resolver.php

```diff
@@ -162,6 +187,33 @@ public static function get_core_data() {
 		return static::$core;
 	}
 
+	/**
+	 * Checks whether the registered blocks are the same
+	 * as the ones we already processed for this origin.
```

Comment:
Avoid "we" in comments.

```suggestion
	 * Checks whether the registered blocks were already processed for this origin.
```